src/components/FeatureViews/AdminPortal/UnpaidSessions.js

```diff
@@ -0,0 +1,36 @@
+import React, { useMemo } from 'react';
+import {useDispatch, useSelector} from "react-redux";
+import {bindActionCreators} from "redux";
+import Loading from "components/Loading";
+import * as adminActions from "../../../actions/adminActions";
+import { useEffect } from 'react';
+import UnpaidSessionCard from './UnpaidSessionCard';
+import * as hooks from 'actions/hooks';
+
+function UnpaidSessions() {
+    const dispatch = useDispatch();
+    const api = useMemo(
+        () => bindActionCreators(adminActions, dispatch),
+        [dispatch]
+    );
+    const UnpaidList = useSelector(({Admin}) => Admin.Unpaid) || [];
+    const studentList = useMemo(()=>  UnpaidList.map(({student})=>student), [UnpaidList])
+    const courseList = useMemo(()=> UnpaidList.map(({course})=>course), [UnpaidList])
+
+    const studentStatus = hooks.useStudent(studentList);
+    const courseStatus = hooks.useCourse(courseList);
+
+    useEffect(()=>{
+        api.fetchUnpaid();
+    },[]);
```

Comment:
Can be replaced with the `useUnpaidSessions` if you fix that.

Reply:
replace all three lines with useUnpaidSessions?